src/core/evaluator.js

```diff
@@ -1640,19 +1651,34 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
                 xObjStateManager.transform(matrix);
               }
 
+              // Enqueue textContent chunk before call for sub-form.
```

Comment:
Nit: What does "sub-form" mean here?
It might be better to just write something along the lines of:
```javascript
// Enqueue the `textContent` chunk before parsing the /Form XObject.
```